components/portal/src/components/image/index.js

```diff
@@ -16,23 +16,34 @@
  * under the License.
  */
 
+/* eslint max-lines: ["off"] */
```

Comment:
Shall we not switch this off ? Let's increase the limit to 600 for this file only

Reply:
Increased the line count.